client/src/components/Auth/RegisterForm.js

```diff
@@ -0,0 +1,395 @@
+        justify="center"
+        alignItems="center"
+      >
+        <Grid
+          container
+          direction={matchMobile ? 'column' : 'row'}
+          justify="flex-start"
+          alignItems="center"
+          lg={5}
+          md={5}
+          xs={12}
+        >
+          <Box className={classesMobile.title}>
+            {
+              matchMobile ? (
+                <Typography
+                  onClick={() => props.loginModal()}
+                  className={classNames(classes.title, classesMobile.inactive)}
+                  style={tungsten}
+                  variant="h3"
+                  color="textPrimary"
+                  align="center"
+                >
+                  Log In
+                </Typography>
+              ) : null
+            }
+            <Typography
+              className={classNames(classes.title)}
+              style={tungsten}
+              variant="h3"
+              color="textPrimary"
+              align="center"
+            >
+              Create Account
+            </Typography>
+          </Box>
+          <FormControl className={classes.formControl} variant="outlined">
+            <InputLabel classes={{ root: classes.label }} htmlFor="email">
+              Email
+            </InputLabel>
+            <OutlinedInput
+              id="email"
+              classes={{
+                root: classes.inputRoot,
+                input: classes.input,
+              }}
+              value={state.loginOrEmail}
+              onChange={onChange('email')}
+              labelWidth={40}
+              fullWidth
+            />
+          </FormControl>
+          <FormControl className={classes.formControl} variant="outlined">
+            <InputLabel classes={{ root: classes.label }} htmlFor="login">
+              Login
+            </InputLabel>
+            <OutlinedInput
+              id="login"
+              classes={{
+                root: classes.inputRoot,
+                input: classes.input,
+              }}
+              value={state.login}
+              onChange={onChange('login')}
+              labelWidth={40}
+              fullWidth
+            />
+          </FormControl>
+          <FormControl className={classes.formControl} variant="outlined">
+            <InputLabel classes={{ root: classes.label }} htmlFor="password">
+              Password
+            </InputLabel>
+            <OutlinedInput
+              id="password"
+              type={passwordHidden ? 'password' : 'text'}
+              classes={{
+                root: classes.inputRoot,
+                input: classes.input,
+              }}
+              value={state.password}
+              onChange={onChange('password')}
+              labelWidth={75}
+              fullWidth
+              endAdornment={
+                state.password
+                  ? (
+                    <PasswordIcon
+                      onMouseDown={() => setPasswordVisible(false)}
```

Comment:
the same things as was mention in LoginForm.

Reply:
Resolved